### PR TITLE
test: Cypress - Flaky test fix

### PR DESCRIPTION
**Pull request by Vijetha-Kaja**
## Description

**Fixed below flaky tests**

- Fork_Template_spec.js
- PageOnLoad_spec.ts
- False_Spec.ts
- MySQL1_Spec.ts
- Scrolling_spec.ts

## Type of change

- Flaky test fix

## How Has This Been Tested?
- Cypress test runs

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

File: app/client/cypress/integration/Regression_TestSuite/ClientSideTests/Templates/Fork_Template_spec.js [hunk 1]
```diff
@@ -60,6 +60,11 @@ describe("Fork a template to an workspace", () => {
 
   it("4. Check if tooltip is working in 'Reconnect Datasources'", () => {
     cy.NavigateToHome();
+    cy.get("body").then(($ele) => {
+      if ($ele.find(reconnectDatasourceLocators.Modal).length) {
+        cy.get(DataSources._skiptoApplicationBtn).click();
+      }
+    });
```
Comment (Aishwarya-U-R) [inline app/client/cypress/integration/Regression_TestSuite/ClientSideTests/Templates/Fork_Template_spec.js:67]: We have to check the tooltip in reconnect modal only know? Why are you skipping to application here?

File: app/client/cypress/integration/Regression_TestSuite/ClientSideTests/Templates/Fork_Template_spec.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ const commonlocators = require("../../../../locators/commonlocators.json");
 const templateLocators = require("../../../../locators/TemplatesLocators.json");
 const reconnectDatasourceLocators = require("../../../../locators/ReconnectLocators.js");
 import { ObjectsRegistry } from "../../../../support/Objects/Registry";
-const { AggregateHelper, HomePage } = ObjectsRegistry;
+const { AggregateHelper, DataSources, HomePage } = ObjectsRegistry;
```
Comment (Aishwarya-U-R) [inline app/client/cypress/integration/Regression_TestSuite/ClientSideTests/Templates/Fork_Template_spec.js:5]: Import from ObjectsCore.

File: app/client/cypress/integration/Regression_TestSuite/ClientSideTests/Templates/Fork_Template_spec.js (hunk 1 above)
Comment (Vijetha-Kaja) [reply app/client/cypress/integration/Regression_TestSuite/ClientSideTests/Templates/Fork_Template_spec.js:67]: I have added this to make it work on 2nd attempt. If the first attempt fails, reconnect modal appears before forking the template.

File: app/client/cypress/support/Pages/DataSources.ts [hunk 3]
```diff
@@ -767,6 +767,7 @@ export class DataSources {
       .scrollIntoView()
       .should("be.visible")
       .closest(this._datasourceCard)
+      .scrollIntoView()
```
Comment (Aishwarya-U-R) [inline app/client/cypress/support/Pages/DataSources.ts:770]: Is this for Scrolling_spec fix?

**Review by Aishwarya-U-R — APPROVED** [review]

**Pull request merged**